config/application.rb

```diff
@@ -22,8 +22,14 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
-    
+
     # Gzip files when possible
     config.middleware.use Rack::Deflater
+
+    #HCaseflow application urls
+    config.app_urls = {"Caseflow" => "https://caseflow.ds.va.gov", "Efolder" => "https://efolder.cf.ds.va.gov"}
```

Comment:
i think that's worse from the integration side. that means your clients will have to redirect to `www.feedback.caseflow.ds.va.gov?app=1`. harder to remember, less readable url. why do you think that option is better?